components/cacheSettings/index.js

```diff
@@ -1,4 +1,4 @@
-import { Container, RadioGroup } from "@newfold/ui-component-library";
+import { Container, RadioGroup, Alert } from "@newfold/ui-component-library";
```

Comment:
I don't see `Alert` used in this file.

Reply:
I removed the unused component, it's committed